fernandolobato/balarco
# Incorporated token authentication in REST controller

Added the IsAuthenticated permission class into the Contact REST controller and modified the tests to force_authenticate using Tokens :)

#### What's this PR do?
It adds an authentication class to the Contact REST controller so that it uses token authentication implemented previously.

#### Where should the reviewer start?
clients/views.py
clients/test_views.py

#### How should this be manually tested?
Once should run the app with:

`python manage.py runserver
`

browse to: 

http://127.0.0.1:8000/clients/api-contact/

The controller should not allow acces.

login at: 

http://127.0.0.1:8000/users/api-login

test again

#### Any background context you want to provide?



This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)
